tests/system/aiplatform/test_e2e_tabular.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import importlib
+import pytest
+import os
+import uuid
+
+from urllib import request
+
+from google.cloud import aiplatform
+from google.cloud import storage
+from google.cloud.aiplatform import initializer
+
+
+_PROJECT = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+_LOCATION = "us-central1"
+
+_BLOB_PATH = "california-housing-data.csv"
+_DATASET_SRC = "https://dl.google.com/mlcc/mledu-datasets/california_housing_train.csv"
+_DIR_NAME = os.path.dirname(os.path.abspath(__file__))
+_LOCAL_TRAINING_SCRIPT_PATH = os.path.join(
+    _DIR_NAME, "test_resources/california_housing_training_script.py"
+)
+_TEMP_PREFIX = "temp-vertex-sdk-e2e-tabular"
+
+
+class TestEndToEndTabular:
+    """End to end system test of the Vertex SDK with tabular data adapted from
+    reference notebook http://shortn/_eyoNx3SN0X"""
+
+    def setup_method(self):
+        importlib.reload(initializer)
+        importlib.reload(aiplatform)
+
+    @pytest.fixture()
+    def shared_state(self):
+        shared_state = {}
+        yield shared_state
+
+    @pytest.fixture()
+    def prepare_staging_bucket(self, shared_state):
+        """Create a staging bucket, download CSV data, and copy into bucket."""
+
+        new_staging_bucket = f"{_TEMP_PREFIX}-staging-{uuid.uuid4()}"[:63]
+        shared_state["staging_bucket"] = new_staging_bucket
+
+        storage_client = storage.Client(project=_PROJECT)
+        shared_state["storage_client"] = storage_client
+
+        bucket = storage_client.create_bucket(new_staging_bucket, location=_LOCATION)
```

Comment:
What are the advantages of creating a new bucket for each integration test vs using a common bucket and creating a new directory?